site/content/overview/custom-policies.md

```diff
@@ -0,0 +1,159 @@
+| [ObservabilityPolicy]({{<relref "/how-to/monitoring/tracing.md" >}}) | Define settings related to tracing, metrics, or logging | Direct          | HTTPRoute, GRPCRoute          | Yes                           | No        | v1alpha1    |
+
+
+{{< important >}}
+NGINX Gateway Fabric policies do not work with [HTTPRoute matches](https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io/v1.HTTPRouteMatch) with `headers`, `params`, or `method` matchers defined. This will be addressed in a future release.
+{{< /important >}}
+
+## Terminology
+
+- _Attachment Type_. How the policy attaches to an object. Attachment type can be "direct" or "inherited".
+- _Supported Target Object(s)_. API objects the policy can be applied to.
+- _Supports Multiple Target Refs_. Whether a single policy can target multiple objects.
+- _Mergeable_. Whether policies that target the same object can be merged.
+
+## Direct Policy Attachment
+
+A Direct Policy Attachment is a policy that references a single object -- such as a Gateway or HTTPRoute. It is tightly bound to one instance of a particular Kind within a single Namespace or an instance of a single Kind at the cluster-scope. It affects _only_ the object specified in its TargetRef.
+
+This diagram uses a fictional retry policy to show how Direct Policy Attachment works:
+
+{{<img src="img/direct-policy-attachment.png" alt="Direct Policy Attachment">}}
+
+The policy targets the HTTPRoute `baz` and sets `retries` to `3` and `timeout` to `60s`. Since this policy is a Direct Policy Attachment, its settings are only applied to the `baz` HTTPRoute.
+
+## Inherited Policy Attachment
+
+Inherited Policy Attachment is designed to allow settings to flow down a hierarchy. The hierarchy for Gateway API resources looks like this:
+
+{{<img src="img/hierarchy.png" alt="Hierarchy">}}
+
+Settings defined in a policy attached to an object in this hierarchy may be inherited by the resources below it. For example, the settings defined in a policy attached to a Gateway may be inherited by all the HTTPRoutes attached to that Gateway.
+
+Settings in an Inherited Policy can be Defaults or Overrides. Defaults set the default value for something and can be overridden by policies on a lower object. Overrides cannot be overridden by lower objects.
+All settings in NGINX Gateway Fabric Inherited Policies are Defaults.
+
+Default values are given precedence from the bottom up. Therefore, a policy setting attached to a Backend will have the highest precedence over the one attached to higher objects.
+
+The following diagram shows how Inherited Policies work in NGINX Gateway Fabric using a fictional retry policy:
+
+{{<img src="img/inherited-policy-attachment.png" alt="Inherited Policy Attachment">}}
+
+There are three policies defined:
+
+- Blue policy that targets the `dev` Gateway
+- Orange policy that targets the `baz` HTTPRoute
+- Pink policy that targets the `foo` HTTPRoute
```

Comment:
Not sure our policy on including colors in the docs (thinking about color-blindness).

Reply:
Good call. I'll name the policies instead